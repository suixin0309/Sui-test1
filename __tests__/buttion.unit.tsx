import * as React from "react"
import * as renderer from "react-test-renderer"
import Button from "../lib/button"

describe('button用例', () => {
    it('是div', () => {
        const json=renderer.create(<Button/>).toJSON()
        expect(json).toMatchSnapshot()
    });
});