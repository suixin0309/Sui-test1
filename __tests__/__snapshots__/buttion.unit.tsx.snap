// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`button用例 是div 1`] = `
<div>
  button
</div>
`;
